src/app/pages/forms/components/layouts/components/imageCropper/index.ts

```diff
@@ -0,0 +1 @@
+export * from './imageCropper.component';
```

Comment:
Please, don't create new `index.ts`. Use the full path to a needed file instead of 'index.ts'.